src/main/java/org/testng/util/TimeUtils.java

```diff
@@ -22,4 +25,37 @@ public static String formatTimeInLocalOrSpecifiedTimeZone(
     sdf.setTimeZone(timeZone);
     return sdf.format(timeInMilliSeconds);
   }
+
+  /**
+   * A sample task to be executed.
+   */
+  @FunctionalInterface
+  public interface Task {
```

Comment:
You can use `Runnable` otherwise

Reply:
I thought I would use a specific interface that is not `Runnable` because that usually gives the reader an impression that we are doing something with threads, which is not the case.